### PR TITLE
Add support for Scala 2.13.2

### PR DESCRIPTION
**Pull request by tgodzik**
Also bumps scalameta to 4.3.9 and Bloop to the newest snapshot.

### CODE REVIEW

**Comment by kpbochenek** [comment]
```
[error] 15:  test("future-single") {
[error] 16:    assertDiffEqual(
[error] 17:      Messages.FutureScalaVersion.message(Seq("2.13.50")),
[error] => Obtained
[error] "You are using Scala version 2.13.50, which is not yet supported in this version of Metals. Please downgrade to Scala version 2.13.2 for the moment until the new Metals release."
```

It might be worth to update test not to check hardcoded version, we won't hit on this problem in next releases.

File: mtags/src/main/scala-2/scala/meta/internal/pc/ScalaPresentationCompiler.scala [hunk 1]
```diff
@@ -34,7 +34,7 @@ import scala.meta.internal.metals.FoldingRangeProvider
 import scala.meta.internal.metals.Trees
 import org.eclipse.lsp4j.DocumentSymbol
 import scala.meta.internal.metals.DocumentSymbolProvider
-import org.eclipse.lsp4j.{DocumentOnTypeFormattingParams, TextEdit}
```
Comment (tgodzik) [inline mtags/src/main/scala-2/scala/meta/internal/pc/ScalaPresentationCompiler.scala:37]: The import was actually repeated and this only got reported with Scala 2.13.2 :tada:

File: mtags/src/main/scala/scala/meta/internal/metals/WorkspaceSymbolQuery.scala [hunk 2]
```diff
@@ -42,7 +42,7 @@ object WorkspaceSymbolQuery {
   def fromTextQuery(query: String): WorkspaceSymbolQuery = {
     val isTrailingDot = query.endsWith(".")
     val isClasspath = query.contains(";")
-    val actualQuery = query.stripSuffix(".").replaceAllLiterally(";", "")
```
Comment (tgodzik) [inline mtags/src/main/scala/scala/meta/internal/metals/WorkspaceSymbolQuery.scala:45]: This is actually deprecated, started poping up with 2.13.2.

File: build.sbt [hunk 3]
```diff
@@ -546,6 +544,7 @@ lazy val mtest = project
       "scala3" -> V.scala3,
       "scalaVersion" -> scalaVersion.value
     ),
+    crossScalaVersions := V.nonDeprecatedScalaVersions,
```
Comment (tgodzik) [inline build.sbt:547]: This caused issues with a workflow like:
```
++2.13.2
cross/test
```

File: build.sbt [hunk 4]
```diff
@@ -308,7 +302,11 @@ val mtagsSettings = List(
       crossSetting(
         scalaVersion.value,
         if211 = List("com.lihaoyi" %% "pprint" % "0.5.4"),
-        otherwise = List("com.lihaoyi" %% "pprint" % "0.5.9")
+        if2 = List("com.lihaoyi" %% "pprint" % "0.5.9"),
```
Comment (tgodzik) [inline build.sbt:305]: Fixes pprint to be available in mtags. Otherwise branch was never hit.

**Pull request merged**